### ISSUE TITLE
ERROR conda.core.link:_execute(507): An error occurred while installing package 'conda-forge::async_generator-1.10-py_0'

### ISSUE THREAD

**Issue by Rafiur** [polimi-ispl/icpr2020dfdc#57]
While creating the environment this error occurs and rolls back the creation of all other libraries. How do I get past this?
Also if I wanted to train my own dataset what is the pipeline?

**Comment by CrohnEngineer** [comment]
Hey @Rafiur ,

thanks for passing by!
Seems like you're trying to install as a package "conda-forge", which instead is a channel for downloading packages.
To replicate our environment just clone our repo, and then from the bash execute the following lines:
```bash
$ conda env create -f environment.yml
$ conda activate icpr2020

```
On what kind of dataset do you want to train on?
Please be aware that this is a repository for replicating the results of a scientific paper, so the pipeline is tailored to the two datasets we have used (i.e., Deep Fake Detection Challenge and FaceForensics++).
Bests,

Edoardo

**Comment by Rafiur** [comment]
The error still occurs when I run "conda env create -f environment.yml" on ananconda prompt
<img width="960" alt="error_1" src="https://user-images.githubusercontent.com/42866452/124922641-81b82580-e01b-11eb-9d75-deb71d7a7a76.PNG">

yes I have seen that the repository is tailored for two datasets but I was hoping to know the preprocessing procedure of the dataset and how to do that

**Comment by nicobonne** [comment]
Hey @Rafiur, 
I just managed to recreate the environment from scratch under Ubuntu 20.10, so I think the error is related to Windows specific packages managed by Anaconda. I don't have a solution for you, try updating/reinstalling Anaconda hoping for an improvement. In general, I saw many people here using Windows without this problem, so my guess is that there is something broken in your Anaconda configuration. Maybe @CrohnEngineer has some Windows specific hints to share. 

Regarding the training with your dataset: we don't cover this in our scripts. If you are really motivated in doing so, the steps to follow are:
1) indexing videos by modifying one of the index scripts [index_dfdc.py](https://github.com/polimi-ispl/icpr2020dfdc/blob/master/index_dfdc.py) or [index_ffpp.py](https://github.com/polimi-ispl/icpr2020dfdc/blob/master/index_ffpp.py) 
2) extracting faces by modifing [extract_faces.py](https://github.com/polimi-ispl/icpr2020dfdc/blob/master/extract_faces.py)

then it _should_ be straightforward using one of the training scripts and test them accordingly.
